server/middleware/setUpHealthChecks.ts

```diff
@@ -1,8 +1,10 @@
 import express, { Router } from 'express'
 
 import healthcheck from '../services/healthCheck'
+import { Services } from '../services'
+import rolledOutPrison from '../services/rolledOutPrision'
 
```

Comment:
Spelling `rolledOutPrision` and convention if this is a service, name it like `rolledOutPrisonService` please?

Reply:
Sorry, my bad with spelling. Corrected and also renamed service.